utilities/virt.py

```diff
@@ -132,7 +113,12 @@ def wait_for_vm_interfaces(vmi: VirtualMachineInstance, timeout: int = TIMEOUT_1
         TimeoutExpiredError: After timeout reached.
     """
     # Waiting for guest agent connection before checking guest agent interfaces report
-    wait_for_guest_agent(vmi=vmi, timeout=timeout)
+    LOGGER.info(f"Wait until guest agent is active on {vmi.name}")
+    vmi.wait_for_condition(
```

Comment:
If you already implemented `wait_for_agent_connected` below - why not calling it instead of explicitly checking the condition?

Reply:
That one is a method (of the VM class) while this one is a function that works with a VMI.